src/main/kotlin/com/dp/advancedgunnerycontrol/typesandvalues/WeaponAITags.kt

```diff
@@ -10,7 +10,7 @@ import com.fs.starfarer.api.Global
 import com.fs.starfarer.api.combat.WeaponAPI
 import com.fs.starfarer.api.fleet.FleetMemberAPI
 
-val pdTags = listOf("PD", "NoPD", "PD(Flx>N%)")
+val pdTags = listOf("PD", "NoPD", "PD(Flx>N%), PrioritisePD")
```

Comment:
I feel like you are missing a pair of quotation marks here

Reply:
Well that is embarrassing xD